wcc-contentful/lib/wcc/contentful/event_emitter.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module WCC::Contentful::EventEmitter
```

Comment:
could you use this library instead of reinventing the paradigm?

https://github.com/krisleech/wisper

Reply:
I'm not opposed to it, but I have an aversion to using "micro" libraries when it's something I'm capable of implementing in one file.  My mind returns to the left-pad fiasco.

Also, that library is > 10 files, hardly "micro".